iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -194,6 +203,88 @@ public void testScanTable() throws IOException {
     Assert.assertArrayEquals(new Object[] {"Alice", 0L}, descRows.get(2));
   }
 
+  @Test
+  public void testMigrateHiveTableToIceberg() {
+    Assume.assumeTrue("migration is only supported for hive catalog",
```

Comment:
What happens if different catalog is configured?

Reply:
It works  :) (with a bit of tweaking)